fe/src/components/Tab/ListingTab.vue

```diff
@@ -1,77 +1,70 @@
 <template>
-    <v-app class="pl-10">
       <v-navigation-drawer permanent>
         <v-list dense>
           <v-list-item-group>
   
             <!--Listing-->
-            <v-list-item href="#listing" :class="{'option-active' : changeActive('listing')}">
+            <v-list-item  @click.capture="handleClick(0)" :class="{ 'option-active': activeIndex === 0 }">
```

Comment:
@LennardUy104  instead of indexes, will it be okay to have some descriptive parameters like the actual name of the tabs?

Reply:
sure maam